deployment.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: database
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: database
+  name: mysql-deployment
+  labels:
+    app: mysql
+spec:
+  replicas: 3
```

Comment:
Remember next time that you need a PV or something else to sync data between mysql replicas